openedx.yaml

```diff
@@ -0,0 +1,7 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
```

Comment:
The file was renamed, so the current URL for this should be http://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification